pkg/project/util.go

```diff
@@ -175,3 +178,25 @@ var intersection = apply(func(bs []Equatable, a Equatable) bool {
 var difference = apply(func(bs []Equatable, a Equatable) bool {
 	return !contains(bs, a)
 })
+
+var getBoolEnvVar = func(key string, fallback bool) (bool, error) {
+	value, exists := os.LookupEnv(key)
+	if exists {
+		boolValue, err := strconv.ParseBool(value)
+		if err == nil {
+			return boolValue, nil
+		} else {
+			return fallback, fmt.Errorf("WARN: environment value of %s should be of type Boolean. Setting to a fallback value %t", key, fallback)
+		}
+	}
+	return fallback, nil
+}
+
+func containsInt(ints []int, i int) bool {
```

Comment:
For his case, he could literally use a switch/case statement. Creating a function is a good idea when it saves code. Since it's used exactly once, at best it's a wash.